modules/unauthenticated-users-cluster-role-binding.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * authentication/impersonating-system-admin.adoc
+// * cicd/builds/triggering-builds-build-hooks.adoc
+// * authentication/tokens-scoping.adoc
+// * authentication/managing-oauth-access-tokens.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-cluster-role-bindings_{context}"]
+= Adding unauthenticated groups to cluster role bindings
+
+As a cluster administrator, you can add unauthenticated users to the following cluster role bindings in {product-title}. Unauthenticated users do not have access to most cluster role bindings by default.
+
+* `system:scope-impersonation`
+* `system:webhook`
+* `system:oauth-token-deleter`
+* `self-access-reviewer`
```

Comment:
We should create individual entries for all 4 cases. A combination of those 4 cases could be an entry that helps administrators to have the exact same setup as before, but for, e.g. in the case of [Build triggers](https://docs.openshift.com/container-platform/4.15/cicd/builds/triggering-builds-build-hooks.html#builds-webhook-triggers_triggering-builds-build-hooks), the only cluster role of interest is `system:webhook`. Showing all 4 of them adds a lot of confusion.